### PR TITLE
A few minor changes based on issues noted at https://confluence.lsstcorp.org/pages/viewpage.action?pageId=102468418.

### PR DESCRIPTION
**Pull request by jeffcarlin**

### CODE REVIEW

File: LDM-554.tex [hunk 1]
```diff
@@ -744,8 +744,9 @@ \subsubsection{General Tabular Queries}
...
+Lists of positions may be uploaded or taken from the Workspace.  Efficient implementation of list-based queries requires a corresponding API aspect / Data Access Web API service, to avoid the submission of large numbers of separate queries.
+The system is not required to, but may, accept lists in which different elements are specified in more than one of the supported means of specifying positions; e.g., it is not required to support a list that is partially in equatorial coordinates and partially in LSST object IDs, because of the parsing and interpretation complexities involved.  The implementation should not preclude adding the ability to mix specification types later on, guided by feedback from the user community as to what is most useful.
+Specific coordinates required elsewher in this document include "equatorial, ecliptic or galactic coordinates" in DMS-PRTL-REQ-0020 and DMS-PRTL-REQ-0022, "astrophysical source name" (DMS-PRTL-REQ-0023), "LSST catalog entry identifiers" (DMS-PRTL-REQ-0024), and "external Solar System Object identifiers" (DMS-PRTL-REQ-0025).
```
Comment (leannep) [inline LDM-554.tex]: Specific coordinates required elsewhere

**Review by leannep — CHANGES_REQUESTED** [review]
Only one spelling mistake - good to go

**Comment by jdswinbank** [comment]
Sorry if I'm just being wrong and/or confusing here but: isn't this document [autogenerated from MagicDraw](https://github.com/lsst/LDM-554/blob/master/LDM-554.tex#L11)? If so, you need to make changes there, not on GitHub.

**Comment by jeffcarlin** [comment]
The `jira_sync` branch is autogenerated, but not `master`.

File: LDM-554.tex (hunk 1 above)
Comment (jeffcarlin) [reply LDM-554.tex]: Fixed the "elsewher"...

**Comment by leannep** [comment]
> Sorry if I'm just being wrong and/or confusing here but: isn't this document [autogenerated from MagicDraw](https://github.com/lsst/LDM-554/blob/master/LDM-554.tex#L11)? If so, you need to make changes there, not on GitHub.

And only the jira_docugen.tex file in the jira-sync branch

**Comment by jdswinbank** [comment]
Uh... [there is no `jira_sync` branch](https://github.com/lsst/LDM-554/branches/all).

And `master` doesn't contain a `jira_docugen.tex` file.

But hey, I'll butt out now. :-)

**Comment by jdswinbank** [comment]
For what it's worth, I'm told by @timj:

> You are right about LDM 554. It’s a requirements document not a test plan so source of truth is magic draw.  I’m on phone so can’t comment on github at the moment.
> All the changes they are making need to be transferred to magic draw and the document regenerated.

So much for butting out. :-)

**Comment by leannep** [comment]
> For what it's worth, I'm told by @timj:
> 
> > You are right about LDM 554. It’s a requirements document not a test plan so source of truth is magic draw.  I’m on phone so can’t comment on github at the moment.
> > All the changes they are making need to be transferred to magic draw and the document regenerated.
> 
> So much for butting out. :-)

Sorry, I got confused and  thought this was LDM-540. LDM-544 is a requirements document. The branch was created to propose and review changes before updating in MD

**Comment by ktlim** [comment]
The process is well-documented at https://developer.lsst.io/project-docs/change-controlled-docs.html#requirements-documents
It does not include making changes to master directly.  Modifying generated documents without modifying the source of truth (MagicDraw) has gotten us into big trouble in the past in Systems Engineering (changes not actually implemented, implemented differently, implemented incompletely, etc.), so we've been trying to avoid that at the DM level.

**Comment by timj** [comment]
Submit the RFC using the deployed version at https://ldm-554.lsst.io  -- once approved we can work on migrating the fixes to Magicdraw and regenerating the latex from there.

**Pull request merged**